packages/firebase_core/firebase_core_platform_interface/lib/src/firebase_exception.dart

```diff
@@ -30,9 +30,9 @@ class FirebaseException implements Exception {
   FirebaseException({
     required this.plugin,
     this.message,
-    this.code = 'unknown',
+    String? code,
     this.stackTrace,
-  });
+  }), this.code = code ?? 'unknown';
```

Comment:
```suggestion
  }): this.code = code ?? 'unknown';
```
You need to use a colon for invoking an initializer list.

Reply:
@hyungtaecf